lib/ui/compositing.dart

```diff
@@ -712,18 +721,27 @@ class SceneBuilder extends NativeFieldWrapperClass1 {
   /// Adds a [Picture] to the scene.
   ///
   /// The picture is rasterized at the given offset.
-  void addPicture(
+  ///
+  /// {@macro dart.ui.sceneBuilder.oldLayer}
+  ///
+  /// {@macro dart.ui.sceneBuilder.oldLayerVsRetained}
+  PictureEngineLayer addPicture(
     Offset offset,
     Picture picture, {
     bool isComplexHint = false,
     bool willChangeHint = false,
+    PictureEngineLayer? oldLayer,
```

Comment:
I'm not sure if `oldLayer` is needed in this place, it doesn't look useful. Probability it makes more sense not to add it?